src/lib/core/dataStorage.ts

```
@@ -43,16 +43,33 @@ export const AUTOMATION_INITIAL_DATA: AutomationSettings = [];
-    dataDirectory?: string;
     accountsFile?: AccountDataFile;
     settingsFile?: AllSettings;
     deviceAuthsFile?: DeviceAuthsSettings;
     automationFile?: AutomationSettings;
     taxiFile?: TaxiSettings;
+    downloaderFile?: DownloaderSettings;
   } = {};
 
+  static async init() {
+    const home = await homeDir();
+    DOWNLOADER_INITIAL_DATA.downloadPath = DOWNLOADER_INITIAL_DATA.downloadPath!.replace('%HOME%', home);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)